src/resdk/resources/__init__.py

```diff
@@ -41,6 +41,14 @@
 .. autoclass:: resdk.resources.DescriptorSchema
    :members:
    :inherited-members:
+   
+.. autoclass:: resdk.resources.AnnotationValue
```

Comment:
consider adding the AnnotationGroup as well, as it is used, e.g. in the `AnnotationField.group`

Reply:
Sure.